src/api/browse.ts

```diff
@@ -8,8 +8,21 @@ import { FacetFilterModel } from "../models/facets";
 import { fetchJson } from "../utils/utils";
 import { showMessage } from "../components/messages/usage";
 
-const MASS_URL = process.env.REACT_APP_MASS_URL;
-const METLDATA_URL = process.env.REACT_APP_METLDATA_URL;
+/**
+ * Checks and cleans up URL if last character is or is not forward slash
+ * Duplicate of utils version because of infinite recursion error, needs refactoring
+ * @param url - String of URL
+ * @param endSlash - whether we want to have an end slash or not
+ * @returns URL as specified
+ */
+export const urlWithEndSlash = (url: string) => {
+  const lastCharSlash : boolean = url.slice(-1) === "/"
```

Comment:
```suggestion
  const lastCharSlash = url.endsWith("/");
```

type an be inferred again here

Reply:
or maybe just
```js
return url.endsWith("/") ? url  : url + "/";
```

as you see the functions gets much simpler when not using boolean params :)